integrated_channels/integrated_channel/models.py

```diff
@@ -205,6 +206,76 @@ def clean(self):
                 }
             )
 
+    def get_learner_data_audit_model(self):
+        return LearnerDataTransmissionAudit.get_completion_class_by_channel_code(self.channel_code())
+
+    def get_last_learner(self, error, config_id):
+        """
+        Gets every learner audit associated with the configuration and finds the most recent record.
```

Comment:
this docstring should say that it finds the most recent _timestamp_ (or `None`) because thats what it returns - the current docstring makes it seem like you'll return the learner audit record most recently modified